packages/webdriverio/src/commands/element/getCSSProperty.ts

```diff
@@ -83,66 +77,78 @@ export async function getCSSProperty (
     cssProperty: string,
     pseudoElement?: PseudoElement,
 ) {
+    const browser = getBrowserObject(this)
+
     if (cssShorthandProps.isShorthand(cssProperty)) {
         const cssValue = await getShorthandPropertyCSSValue.call(
             this,
-            cssProperty,
-            pseudoElement,
+            {
+                cssProperty,
+                pseudoElement,
+                browser
```

Comment:
No need to pass this along since you can call `const browser = getBrowserObject(this)` in `getShorthandPropertyCSSValue`

Reply:
uptated - silly mistake by me sorry 😞